VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,50 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again, and welcome to a new video about AsyncAPI. In this video, we're going to talk about:
+
+AsyncAPI and OpenAPI
+
+•	What is OpenAPI?
+•	Similarities between AsyncAPI and OpenAPI
+•	Differences between them
+•	Example of each initiative
+
+Here we go! 
+
+## What is OpenAPI?
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to OpenAPI. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, or Microsoft. The Linux Foundation also supports this project. 
+
+## Similarities between AsyncAPI and OpenAPI
+
+Its ultimate goal is to work with EDA architectures as simply as with Rest APIs. This specification for the asynchronous world is analogous to OpenAPI in the synchronous world. Both initiatives are based on the same principles:
+
+- Their use is intended for a contract-first philosophy.
+
+- They are implementation agnostic.
+
+- Both use the same syntax. 
+
+- Like AsyncAPI, OpenAPI also offers a set of tools around it.
+
+- They protect information consumption using different security schemes such as user and password, API keys, certificates, and of course, the most recommended ones such as OAuth or OIDC flows.
+
+- Both specifications are extensible by including new properties with the prefix x-.
+
+## Differences between them
+
+As we have seen, the two initiatives are very similar but cover very different fields. REST APIs have a specific function that requires their methods and documentation approaches. REST APIs have, among others, a great solution in the form of OpenAPI, a set of standardized documentation methods for enabling tools and interactivity.
+
+However, message-based APIs are very different and need solutions covering the caveats and considerations intrinsic to message-based architectures. AsyncAPI's tagline says it best: "REST APIs have OpenAPI. Messaging has AsyncAPI."
+
+It's essential to understand that while the two solutions have a lot in common, they are also wildly divergent. Both OpenAPI and AsyncAPI allow for more significant cooperation and collaboration between different documented APIs. Still, OpenAPI is a solution for REST, and AsyncAPI is a solution for message-based architectures. Each is aimed at a different domain.
```

Comment:
```suggestion
 As we have seen, the two initiatives are significantly similar, but they also cover significantly different fields. REST APIs have a specific function that requires their methods and documentation approaches. REST APIs have, among others, a great solution in the form of OpenAPI, a set of standardized documentation methods for enabling tools and interactivity.
 
 On the other hand, message-based APIs are different because they require solutions for the caveats and intrinsic considerations of message-based architectures. AsyncAPI's tagline says it best: "REST APIs have OpenAPI. Messaging has AsyncAPI."
 
 It's essential to understand that while the two solutions have a lot in common, they are also wildly divergent. Both OpenAPI and AsyncAPI allow for more significant cooperation and collaboration between different documented APIs. Still, OpenAPI is a solution for REST, and AsyncAPI is a solution for message-based architectures. Each aims to serve a different domain.
```